frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -203,7 +206,7 @@ export const createOrUpdateBuildConfig = (
     default:
       buildStrategyData = {
         sourceStrategy: {
-          env: [...env, ...customBuildEnvs],
+          env: [...buildEnvs, ...customEnvs],
```

Comment:
As discussed in slack (only as info for other reviews) that the merge here doesn't allow the user to unset the "Start command" at the moment. It is automatically restored from the collapsed environment variables.

Reply:
Updated to hide envs from build config section, and to not pick emptied fields.

Reply:
I personally would use the customEnvs first and then override it with user events, but that's a big IMHO and I wouldn't add this comment if I didn't find the "not saving" issue above. :)